xp/identityserver/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# escape=`
+# Stage 0: prepare files
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} AS prepare
+
+ARG CONFIG_PACKAGE
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ADD files/$CONFIG_PACKAGE /Files/
+
+RUN Expand-Archive -Path /Files/$Env:CONFIG_PACKAGE -DestinationPath /Files/Config 
+
+ # Stage 1: create actual image
+FROM ${BASE_IMAGE}
+
+ARG SQL_USER="sa"
+ARG SQL_SA_PASSWORD
+ARG SQL_DB_PREFIX
+ARG SQL_SERVER="mssql"
+ARG SITE_NAME="identity"
+ARG SITECORE_SITE_NAME="sitecore"
+ARG IDENTITYSERVER_PACKAGE
+
+SHELL ["powershell", "-NoProfile", "-Command", "$ErrorActionPreference = 'Stop';"] 
+
+COPY xp/license/license.xml /Files/
```

Comment:
@Barsonax shouldn't this be `COPY dummylicense.xml /Files/license.xml` (+ logic to use copy the real license at runtime)?